functions/menu.bash

```diff
@@ -176,6 +178,8 @@ show_main_menu() {
     "67 | Log Viewer"             "The openHAB Log Viewer webapp (frontail) " OFF \
     "68 | FireMotD"               "Configure FireMotD to present a system overview on SSH login (optional) " OFF \
     "69 | Bash&Vim Settings"      "Apply openHABian settings for bash, vim and nano (optional) " OFF \
+    "6A | Use zram"               "Use compressed RAM/disk sync for active directories to avoid SD card corruption" OFF \
+    "   | Remove zram"            "Don't use compressed memory (back to standard Raspbian FS layout)" OFF \
```

Comment:
Just curious, why not following the approach above and use 6B?

Reply:
because it's a SINGLE option just on/off this approach is already used in similar cases

Reply:
Sorry, I had only looked at the diff. Forget about my comment.